modules/example/src/main/scala/example/AnalysisTest.scala

```diff
@@ -49,7 +49,7 @@ object AnalysisTest {
     """.query[PGcircle]
   }
 
-  def update(code: String, name: Int): Update0 =
+  def update(name: String, code: String): Update0 =
```

Comment:
This was not consistent with the table definition itself, so it was corrected and used as a test query.